actionsRouter.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const actions = require("./data/helpers/actionModel");
+const router = express.Router();
+
+router.get("/", async (req, res, next) => {
+  try {
+    const action = await actions.get();
+    if (action) {
+      res.status(200).json(action);
+    } else {
+      res.status(404).json({ message: "list of actions not here! " });
+    }
+  } catch (error) {
+    next(error);
+  }
+});
+
+router.get("/:id", validateActionId(), async (req, res) => {
+  try {
+    res.status(200).json(req.action);
+  } catch (error) {
+    next(error);
+  }
+});
+
+//POST ACTION DEPENDS ON PROJECT ID
+
+router.post("/", validateAction(), async (req, res, next) => {
+  try {
+    const action = await actions.insert(req.body);
```

Comment:
It is always good to destructure the req.body and bring out only the stuff you need to structure what will be sent to the database